azure_sdk/lib/profile_latest/modules/eventhub_profile_module.rb

```diff
@@ -0,0 +1,168 @@
+      RegenerateAccessKeyParameters = Azure::ARM::EventHub::Api_2017_04_01::Models::RegenerateAccessKeyParameters
+      Destination = Azure::ARM::EventHub::Api_2017_04_01::Models::Destination
+      Sku = Azure::ARM::EventHub::Api_2017_04_01::Models::Sku
+      OperationDisplay = Azure::ARM::EventHub::Api_2017_04_01::Models::OperationDisplay
+      EHNamespaceListResult = Azure::ARM::EventHub::Api_2017_04_01::Models::EHNamespaceListResult
+      AuthorizationRuleListResult = Azure::ARM::EventHub::Api_2017_04_01::Models::AuthorizationRuleListResult
+      EventHubListResult = Azure::ARM::EventHub::Api_2017_04_01::Models::EventHubListResult
+      ErrorResponse = Azure::ARM::EventHub::Api_2017_04_01::Models::ErrorResponse
+      Operation = Azure::ARM::EventHub::Api_2017_04_01::Models::Operation
+      ConsumerGroupListResult = Azure::ARM::EventHub::Api_2017_04_01::Models::ConsumerGroupListResult
+      OperationListResult = Azure::ARM::EventHub::Api_2017_04_01::Models::OperationListResult
+      CheckNameAvailabilityParameter = Azure::ARM::EventHub::Api_2017_04_01::Models::CheckNameAvailabilityParameter
+      AccessKeys = Azure::ARM::EventHub::Api_2017_04_01::Models::AccessKeys
+      CheckNameAvailabilityResult = Azure::ARM::EventHub::Api_2017_04_01::Models::CheckNameAvailabilityResult
+      TrackedResource = Azure::ARM::EventHub::Api_2017_04_01::Models::TrackedResource
+      EHNamespace = Azure::ARM::EventHub::Api_2017_04_01::Models::EHNamespace
+      AuthorizationRule = Azure::ARM::EventHub::Api_2017_04_01::Models::AuthorizationRule
+      Eventhub = Azure::ARM::EventHub::Api_2017_04_01::Models::Eventhub
+      ConsumerGroup = Azure::ARM::EventHub::Api_2017_04_01::Models::ConsumerGroup
+      SkuName = Azure::ARM::EventHub::Api_2017_04_01::Models::SkuName
+      SkuTier = Azure::ARM::EventHub::Api_2017_04_01::Models::SkuTier
+      AccessRights = Azure::ARM::EventHub::Api_2017_04_01::Models::AccessRights
+      KeyType = Azure::ARM::EventHub::Api_2017_04_01::Models::KeyType
+      EntityStatus = Azure::ARM::EventHub::Api_2017_04_01::Models::EntityStatus
+      EncodingCaptureDescription = Azure::ARM::EventHub::Api_2017_04_01::Models::EncodingCaptureDescription
+      UnavailableReason = Azure::ARM::EventHub::Api_2017_04_01::Models::UnavailableReason
+    end
+
+    #
+    # EventHub
+    #
+    class EventHubClass
+      attr_accessor :operations, :namespaces, :event_hubs, :consumer_groups, :configurable, :base_url, :options, :model_classes
+
+      def initialize(configurable, base_url=nil, options=nil)
+        @configurable, @base_url, @options = configurable, base_url, options
+        client = Azure::ARM::EventHub::Api_2017_04_01::EventHubManagementClient.new(configurable.credentials, base_url, options)
+        if(client.respond_to?(:subscription_id))
+          client.subscription_id = configurable.subscription_id
+        end
+        self.operations = Azure::ARM::EventHub::Api_2017_04_01::Operations.new(client)
```

Comment:
Did we discuss this: Why are we doing new on `Operations` class when we already have `client` that contains `operations` initialized?

Reply:
Done